README.md

```diff
@@ -1,129 +1,114 @@
+1. Clone the repository:
 
-```sh
-flyctl volumes create linkding_data --region <your_region> --size <size_in_gb>
-```
+    ```sh
+    git clone https://github.com/fspoettel/linkding-on-fly.git && cd linkding-on-fly
+    ```
 
-Attach the persistent volume to the container by adding a `mounts` section to `fly.toml`.
+2. Login to [`flyctl`](https://fly.io/docs/getting-started/log-in-to-fly/):
 
-```toml
-[mounts]
-  source="linkding_data"
-  destination="/etc/linkding/data"
-```
+    ```sh
+    flyctl auth login
+    ```
 
-### Configure litestream backups
+3. Create a [persistent volume](https://fly.io/docs/reference/volumes/) to store the `linkding` application data:
 
-> ℹ️ If you want to use another storage provider, check litestream's ["Replica Guides"](https://litestream.io/guides/) section and adjust the config as needed.
+    > ℹ️ Fly's free tier includes `3GB` of storage across your VMs. Since `linkding` is very light on storage, a `1GB` volume will be more than enough for most use cases. It's possible to change volume size later. A how-to can be found in the _"Verify Backups / Scale Persistent Volume"_ section below.
 
-Log into B2 and [create a bucket](https://litestream.io/guides/backblaze/#create-a-bucket). Instead of adjusting the litestream config directly, we will add storage configuration to `fly.toml`. In the `env` section, set `LITESTREAM_REPLICA_ENDPOINT` and `LITESTREAM_REPLICA_BUCKET` to your newly created bucket's endpoint and name.
+    ```sh
+    # List available regions via: flyctl platform regions
+    flyctl volumes create linkding_data --region <region code> --size 1
+    ```
 
-Then, create [an access key](https://litestream.io/guides/backblaze/#create-a-user) for this bucket. Add the key to fly's secret store.
+4. Add the `linkding` superuser credentials to fly's secret store:
 
-```sh
-flyctl secrets set LITESTREAM_ACCESS_KEY_ID="<keyId>" LITESTREAM_SECRET_ACCESS_KEY="<applicationKey>"
-```
+    ```sh
+    flyctl secrets set LD_SUPERUSER_NAME="<username>" LD_SUPERUSER_PASSWORD="<password>"
+    ```
 
-### Create a linkding superuser
+5. Copy the [.env.sample](.env.sample) file to `.env`, fill in the values and source them:
 
-You can create the linkding superuser prior to deployment by adding the following to fly's secret store:
+    ```sh
+    cp .env.sample .env
+    # vim .env
+    source .env
+    ```
 
-```sh
-flyctl secrets set LD_SUPERUSER_NAME="<username>" LD_SUPERUSER_PASSWORD="<password>"
-```
+6. Create the [`fly.toml`](https://fly.io/docs/reference/configuration/) from the [template](templates/fly.toml):
 
-### Deploy to fly
+    ```sh
+    envsubst < templates/fly.toml > fly.toml
```

Comment:
I'm a bit hesitant to merge this templating change as-is. This adds a dependency on `envsubst`which is [not installed on macos by default](https://stackoverflow.com/questions/23620827/envsubst-command-not-found-on-mac-os-x-10-8) and not available at all on windows which `flyctl` both supports. Same issue with `source` and `cp` on windows.

I like the idea of a template itself but think that the guide being usable on all OS that flyctl supports is a feature I'd like to keep.

One solution to this could be to add instructions how to copy paste the template manually to this section and move the `.env` substitution into a [`<details>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details). I'd be open to other ideas as well.

Reply:
Hey @fspoettel - Thank you for your feedback. I think you're definitely right. It's a good feature to keep supporting all OS types.

I like your idea of the `<details>` element. The only other thing I can think of is writing a short PowerShell script which could be ran on Windows to perform the template substitution.